src/Apps/Order2/Routes/Checkout/Components/OfferStep/Order2OfferStep.tsx

```diff
@@ -207,18 +207,48 @@ export const Order2OfferStep: React.FC<Order2OfferStepProps> = ({ order }) => {
         </Flex>
       </Box>
 
-      <Box hidden={currentStep !== CheckoutStepState.ACTIVE}>
+      <Box py={2} px={[2, 4]} hidden={currentStep !== CheckoutStepState.ACTIVE}>
```

Comment:
Simplified the offer forms here by putting the box, offer note and submit buttons in the parent component.